### PR TITLE
Implement commit search on deployed builds

### PR DESCRIPTION
**Pull request by gottfrois**
Thought you might be interested. It's just a simple lookup of commit hash or commit message upon the 10 first deployed builds.

Also allow to configure default project, branch and server name through env variables.

### CODE REVIEW

File: src/deploy.coffee [hunk 1]
```diff
@@ -23,6 +29,16 @@
...
+  if process.env.HUBOT_SEMAPHOREAPP_DEFAULT_BRANCH
+    default_branch = process.env.HUBOT_SEMAPHOREAPP_DEFAULT_BRANCH
+  else
+    default_branch = 'master'
+
+  if process.env.HUBOT_SEMAPHOREAPP_DEFAULT_SERVER
+    default_server = process.env.HUBOT_SEMAPHOREAPP_DEFAULT_SERVER
+  else
+    default_server = 'prod'
+
```
Comment (ben) [inline src/deploy.coffee]: More idiomatic :coffee::

```coffee
default_server = process.env.HUBOT_SEMAPHOREAPP_DEFAULT_SERVER || 'prod'
```

**Comment by ben** [comment]
:heart_eyes: There are event tests! This is awesome.

Can you just clean up the niggle above?

**Comment by ben** [comment]
Oh, one more thing: currently you can do these things:

* `deployed <term>`
* `deployed <term> on <project> to <server>`

Could you add a third form that allows you to let the server be the default?

* `deployed <term> on <project>`

**Comment by gottfrois** [comment]
Sure :)

**Comment by ben** [comment]
:+1: Thanks!

**Pull request merged**